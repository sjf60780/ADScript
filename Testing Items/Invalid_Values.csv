COMPUTERNAME,COMPUTERMAC,COMPUTERDESC,COMPUTEROU,DEPLOYMENTSERVER
,1a2b3c4d5e61,Testing,CORE TEST-LABS,SCUF
OLAB-TEST?,1a2b3c4d5e62,Testing,CORE TEST-LABS,SCUF
OLAB<TEST,1a2b3c4d5e63,Testing,CORE TEST-LABS,SCUF
.OLAB-TEST,1a2b3c4d5e64,Testing,CORE TEST-LABS,SCUF
OLAB-LIBR-0105-069,1a2b3c4d5e65,Testing,CORE TEST-LABS,SCUF
System,1a2b3c4d5e66,Testing,CORE TEST-LABS,SCUF
Anonymous,1a2b3c4d5e67,Testing,CORE TEST-LABS,SCUF
LIBR-0105-014,1a2b3c4d5e68,Testing,CORE TEST-LABS,SCUF
OLAB-TEST-001,,Testing,CORE TEST-LABS,SCUF
OLAB-TEST-002,1a'2b'3c'4d'5e'6f,Testing,CORE TEST-LABS,SCUF
OLAB-TEST-003,1a2b3c4d,Testing,CORE TEST-LABS,SCUF
OLAB-TEST-004,1234abc5678def,Testing,CORE TEST-LABS,SCUF
OLAB-TEST-005,1234abcdefgh,Testing,CORE TEST-LABS,SCUF
OLAB-TEST-006,E4 B9 7A EF FA 2B,Testing,CORE TEST-LABS,SCUF
OLAB-TEST-007,1a2b3c4d5e70,Testing,,SCUF
OLAB-TEST-008,1a2b3c4d5e71,Testing,CORE LABS,SCUF
OLAB-TEST-009,1a2b3c4d5e72,Testing,CORE LABS TRIO OVSS,SCUF
OLAB-TEST-010,1a2b3c4d5e73,Testing,CORE WORKSTATIONS,SCUF
OLAB-TEST-011,1a2b3c4d5e74,Testing,CORE TEST-LABS,
OLAB-TEST-012,1a2b3c4d5e75,Testing,CORE TEST-LABS,COE
OLAB-TEST-013,1a2b3c4d5e76,Testing,CORE TEST-LABS,SUCF
