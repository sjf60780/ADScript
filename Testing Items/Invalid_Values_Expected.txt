1) No Name
2) Name has a ?
3) Name has a >
4) Name starts with a .
5) Name is longer than 15 characters
6) Name is System
7) Name is Anonymous
8) Name exists on AD
9) No MAC
10) MAC delimited with unsupported delimiter (')
11) MAC is too short
12) MAC is too long
13) MAC is not hexadec
14) MAC is tied to existing computer (GLAS-0233-FT1)
15) No OU KW
16) OU returns more than one result (CORE LABS)
17) OU returns no result from too many keywords (CORE LABS TRIO OVSS)
18) OU returns no result from invalid access (CORE WORKSTATIONS)
19) No DP
20) DP doesn't exist (COE)
21) DP is mispelled (SUCF)